app/views/documents/show.html.erb

```diff
@@ -51,6 +51,10 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render "documents/show/contents" %>
 
+    <% if @edition.document_type.pre_release? && current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION) %>
```

Comment:
Minor: we only need the user-level permission check here; we're doing [other work to make the rendering specific to the document type](https://trello.com/c/dzFScHVI/1498-disable-inline-file-attachments-when-featured-attachments-are-available).

Reply:
I'm going to leave this check in for now, otherwise we'll be in the confusing situation where, until the "other work" is completed, people with pre_release features will see an attachments section on news and press releases.

I'll add an action to the other card to replace this check.